src/components/ListItem.tsx

```diff
@@ -33,6 +39,6 @@ const styles = StyleSheet.create({
     paddingVertical: variables.contentPadding,
     borderBottomWidth: 1,
     borderBottomColor: colors.gray2,
-    marginLeft: variables.contentPadding,
+    marginHorizontal: variables.contentPadding,
```

Comment:
I'm not sure why we were doing `marginLeft` only instead of `marginHorizontal` here in the first place. This decision was forcing us to have weird styles elsewhere in the app (basically re-adding the right-padding back in), wherever we use `ListItem` in order to compensate. I manually looked through every instance where we use this and ensured that changes were made to keep styles consistent.

The initial reason for doing it this way _may_ have been to allow us to have horizontal rules on the settings menu that extend all the way to the right of the screen, however, with the new designs, the rules have identical horizontal alignments, so I figured it would be fine to change this sorta hacky style.